XamarinCommunityToolkit/Xamarin.CommunityToolkit.csproj

```diff
@@ -102,4 +100,9 @@
       <Compile Include="**\*.tizen.cs" />
       <Compile Include="**\*.tizen.*.cs" />
     </ItemGroup>
+    <ItemGroup>
+      <Compile Update="Extensions\TranslateExtension.shared.cs">
+        <SubType>Code</SubType>
+      </Compile>
+    </ItemGroup>
```

Comment:
I don't think this is needed. Annoying that VS keeps adding stuff like this

Reply:
Done!